ubuntu/ubuntu-22.x/ubuntu-22.04-hpc/install_rccl.sh

```diff
@@ -1,6 +1,20 @@
 #!/bin/bash
 set -ex
 
+#update CMAKE
+pushd /tmp
+CMAKE_VER="3.30.6"
+wget https://github.com/Kitware/CMake/releases/download/v${CMAKE_VER}/cmake-${CMAKE_VER}-linux-x86_64.tar.gz
```

Comment:
Should this cmake install be in a different script?

Reply:
should I add something like install_cmake.sh and call it from `install_utils.sh` or `install_prerequisites.sh` ?

Reply:
Does this cause problems if it's installed before this point in the installation process? If not then yeah it would probably be best to have it install in the install_prerequisites.sh script.